src/newton1d.jl

```diff
@@ -0,0 +1,99 @@
+doc"""`newton1d` performs the interval Newton method on the given function `f`
+with its derivative `f′` and initial interval `x`.
+Optional keyword arguments give the tolerances `epsX` and `epsf`.
+`epsX` is the tolerance on the relative error whereas `epsf` is the tolerance on |f(X)|,
+and a `debug` boolean argument that prints out diagnostic information."""
+
+function newton1d{T}(f::Function, f′::Function, x::Interval{T};
+                    epsX=eps(T), epsf=eps(T), debug=false)
+
+    L = Array{Interval, 1}()
```

Comment:
This is still not concretely typed, since `Interval` is an abstract type.

```
Interval{T}[]
```

Reply:
Ohh okay, thank you!